Lib/test/test_subclassinit.py

```diff
@@ -264,6 +269,8 @@ def __new__(cls, name, bases, namespace, otherarg):
                 self = super().__new__(cls, name, bases, namespace)
                 self.otherarg = otherarg
                 return self
+            def __init__(self, name, bases, namespace, otherarg):
+                super().__init__(name, bases, namespace)
```

Comment:
add `__init__` to prevent `otherarg` from being passed through to `__init_subclass__`